index.html

```diff
@@ -0,0 +1,1117 @@
+          <h3 class="h3-editors-pick">Erectile Dysfunction Linked to Higher Risk of Death, Cardiovascular Disease</h3>
+          <p>Men reporting erectile dysfunction faced a hazard ratio of 1.4 even when testosterone levels were
+            healthy,
+            the study authors state.</p>
+        </div>
+      </div>
+      <div class="col-md-6 col-lg-3 mt-md-3 mt-lg-5">
+        <img
+          src="https://d.newsweek.com/en/full/1578450/queen-horseriding-windsor.webp?w=397&h=265&f=c708e0135c63971600c358e69fc352ca"
+          alt="" class="w-100">
+        <div class="mt-2">
+          <h3 class="h3-editors-pick">Queen Elizabeth II's Life On Lockdown Revealed By Former Press Secretary</h3>
+          <p>Queen Elizabeth II faces months without horse riding due to social distancing rules but will be
+            exercising,
+            her former press secretary says.</p>
+        </div>
+      </div>
+      <div class="col-md-6 col-lg-3 mt-md-3 mt-lg-5">
+        <img
+          src="https://d.newsweek.com/en/full/1577880/stock-image-salt.webp?w=397&h=265&f=80a661282c34f0ffd7dd359acd1aff97"
+          alt="" class="w-100">
+        <div class="mt-2">
+          <h3 class="h3-editors-pick">Too Much Salt May Lower Your Ability to Fight Bacteria</h3>
+          <p>Tests in mice and humans suggest too much salt can dampen the body's response to infection. A healthy
+            diet,
+            sleep, exercise and reducing stress may help the immune system function.</p>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="row col-lg-12 magazine justify-content-center mt-lg-5 mb-lg-3 border-bottom">
+        <div class="icon icon"></div>
+        <h2 class="mt-2 ml-2">FEATURED SLIDESHOWS</h2>
+      </div>
+      <div class="col-md-4 mt-3 mt-md-3">
+        <img
+          src="https://d.newsweek.com/en/full/1560097/tozeur-tunisia.webp?w=397&h=265&f=69a77ef1bd7e21c060a7b7685b05a73d"
+          alt="" class="w-100">
+        <h3 class="h3-featured-slides mt-3">Relive the 'Star Wars' Saga at These Filming Locations</h3>
+      </div>
+      <div class="col-md-4 mt-5 mt-md-3">
+        <img
+          src="https://d.newsweek.com/en/full/1552749/eyjafjallajokull-iceland.webp?w=397&h=265&f=ea1ffc67b32c527ef435529cb5bd2dc6"
+          alt="" class="w-100">
+        <h3 class="h3-featured-slides mt-3">Active Volcanoes Around the World That Could Erupt at Any Moment</h3>
+      </div>
+      <div class="col-md-4 mt-5 mt-md-3">
+        <img
+          src="https://d.newsweek.com/en/full/1547486/lipschutz-living-room-hotel-chelsea.webp?w=397&h=265&f=fb4270bf5fbc2591cc43d24d0377621d"
+          alt="" class="w-100">
+        <h3 class="h3-featured-slides mt-3">Inside the Last Bohemian Apartments of the Storied Hotel Chelsea</h3>
+      </div>
+    </div>
+    <section id="us-sec" class="row border-top mt-5">
+      <h2 class="col-lg-12 h2-bottom">U.S.</h2>
+      <div class="col-md-4 mb-3">
+        <img
```

Comment:
Check the spacing on your categories'  sections. You have a lot of space after each one and before the division border. Try to make them look like the original page.
Yours:
![Screenshot from 2020-04-13 08-16-49](https://user-images.githubusercontent.com/51926937/79123228-4c148600-7d5f-11ea-87b5-236af34f61bb.png)

Orginal:
![Screenshot from 2020-04-13 08-16-39](https://user-images.githubusercontent.com/51926937/79123235-4fa80d00-7d5f-11ea-845a-11ea851d67e0.png)

On medium screens, the elements on the right look cramped (no spacing between images and text) and the font-size is larger on the main articles.
![Screenshot from 2020-04-13 08-27-38](https://user-images.githubusercontent.com/51926937/79123889-c2fe4e80-7d60-11ea-91b0-c261dc8d8fd7.png)

On mobile screens, the three articles on the bottom should occupy 100% width.
![Screenshot from 2020-04-13 08-27-20](https://user-images.githubusercontent.com/51926937/79123918-d5788800-7d60-11ea-8194-e67f712a1c12.png)